velox/functions/sparksql/CMakeLists.txt

```diff
@@ -35,9 +35,12 @@ target_link_libraries(
 set_property(TARGET velox_functions_spark PROPERTY JOB_POOL_COMPILE
                                                    high_memory_pool)
 
+if(${VELOX_ENABLE_AGGREGATES})
```

Comment:
Why is this change? Let's remove. Feel free to open a separate PR with this change along if necessary.